File: backend/src/main/java/tko/model/dto/workout/LikesWorkoutDTO.java
package tko.model.dto.workout;

import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@NoArgsConstructor
@AllArgsConstructor
public class LikesWorkoutDTO {
    private Long id;
    @NotNull
    private Long userId;
    @NotNull
    private Long workoutId;
}

File: android/app/src/main/java/tk/ssau/fitnesstko/model/dto/LikesWorkoutDto.kt
package tk.ssau.fitnesstko.model.dto

/**
 * Информация о лайке пользователя для тренировки.
 * @param id Уникальный идентификатор записи (может быть null для новых объектов).
 * @param userId Идентификатор пользователя, поставившего лайк.
 * @param workoutId Идентификатор тренировки, которой поставлен лайк.
 */
data class LikesWorkoutDto(
    val id: Long?,
    val userId: Long,
    val workoutId: Long
)
